types/enums/ImageType.ts

```diff
@@ -0,0 +1,5 @@
+export enum ImageType {
+    Company = 'company',
+    Profile = 'profile',
+    Cover = 'cover'
+}
```

Comment:
this are the enum counterpart on the BE

<?php

namespace Conkard\Enums;

enum MediaCollectionType: string
{
    case CARD_COVER_PHOTO = 'CARD_COVER_PHOTO';
    case CARD_COMPANY_LOGO = 'CARD_COMPANY_LOGO';
    case CARD_PROFILE_PICTURE = 'CARD_PROFILE_PICTURE';
}

Reply:
Already remove useless enums